src/Lean/Elab/InfoTree.lean

```diff
@@ -70,9 +70,11 @@ structure FieldInfo where
   stx       : Syntax
   deriving Inhabited
 
-/- We store the list of goals before and after the execution of a tactic.
-   We also store the metavariable context at each time since, we want to unassigned metavariables
-   at tactic execution time to be displayed as `?m...`. -/
+/-- The information needed to render the tactic state in the infoview.
+
+    We store the list of goals before and after the execution of a tactic.
+    We also store the metavariable context at each time since, we want to unassigned metavariables
+    at tactic execution time to be displayed as `?m...`. -/
```

Comment:
```suggestion
    We also store the metavariable context at each time since we want metavariables
    unassigned at tactic execution time to be displayed as `?m...`. -/
```